test/rpc/test_rpc_schema.py

```diff
@@ -0,0 +1,221 @@
+"""
+Test RPC schema validation
+"""
+
+from test.rpc.rpc_utils import rpc_call
+from test.shared import PREDEPLOY_ACCOUNT_CLI_ARGS
+
+import pytest
+from starkware.starknet.public.abi import get_selector_from_name
+
+from starknet_devnet.blueprints.rpc.structures.types import RpcErrorCode
+from starknet_devnet.blueprints.rpc.utils import rpc_felt
+
+
+@pytest.mark.usefixtures("run_devnet_in_background")
+@pytest.mark.parametrize(
+    "params",
+    (
+        {
+            "request": {
+                "contract_address": "0x01",
+                "entry_point_selector": rpc_felt(get_selector_from_name("get_balance")),
+                "calldata": [],
+            },
+        },
+        {
+            "block_id": "latest",
+        },
+        {},
+        {
+            "request": {
+                "entry_point_selector": rpc_felt(get_selector_from_name("get_balance")),
+                "calldata": [],
+            },
+            "block_id": "latest",
+        },
+        {
+            "request": {
+                "contract_address": "0x01",
+                "entry_point_selector": rpc_felt(get_selector_from_name("get_balance")),
+                "calldata": "0x1234",
+            },
+            "block_id": "latest",
+        },
+        {
+            "request": {
+                "contract_address": 1324,
+                "entry_point_selector": rpc_felt(get_selector_from_name("get_balance")),
+                "calldata": [],
+            },
+            "block_id": "latest",
+        },
+        {
+            "request": {
+                "contract_address": "0x01",
+                "entry_point_selector": ["0x01", "0x02"],
+                "calldata": [],
+            },
+            "block_id": "latest",
```

Comment:
What about the test with `block_id` as a number? Is it makes sense?

Reply:
It does. I'll add some.